static/html/adminmenu.htm

```diff
@@ -1,15 +1,15 @@
 <div class="menu">
   <h3><u>Menu</u></h3>
-  <p><a href="/">Home</a></p>
-  <p><a href="/commands">Commands</a></p>
-  <p><a href="/stats">Stats</a></p>
-  <p><a href="/history/">Songlist History</a></p>
+  <p><a href="/">• Home</a></p>
```

Comment:
I've looked into the semantic menu, but it kinda ruins the spacing for me. For next commit I've added the menu_item class. Appreciate the help! 😄